code/modules/spells/spell_types/touch_attacks.dm

```diff
@@ -11,17 +11,16 @@
 		charge_counter = charge_max
 		attached_hand = null
 		to_chat(user, "<span class='notice'>You draw the power out of your hand.</span>")
-		return 0
+		return FALSE
 	..()
 
 /obj/effect/proc_holder/spell/targeted/touch/cast(list/targets,mob/user = usr)
 	for(var/mob/living/carbon/C in targets)
 		if(!attached_hand)
 			if(!ChargeHand(C))
-				return 0
-	while(attached_hand) //hibernate untill the spell is actually used
-		charge_counter = 0
-		stoplag(1)
+				return FALSE
+	UNTIL(!attached_hand) //hibernate untill the spell is actually used
+	charge_counter = 0
```

Comment:
This is a behavior change

Reply:
how? the old one is while attached_hand charge_counter = 0 and sle- oh
oh
oh wait
fuck
woops